package.json

```diff
@@ -21,15 +33,15 @@
         "@types/jest": "^27.5.0",
         "@types/node": "^16.4.13",
         "@types/parse": "^2.18.14",
-        "@types/throng": "^5.0.3",
         "@typescript-eslint/eslint-plugin": "^5.22.0",
         "@typescript-eslint/parser": "^5.22.0",
         "babel-eslint": "^10.1.0",
+        "babel-watch": "7.7.0",
         "eslint": "^8.15.0",
         "eslint-plugin-import": "^2.26.0",
         "jest": "^28.1.0",
+        "nodemon": "^2.0.16",
         "nyc": "^15.1.0",
-        "parse": "^3.4.2",
```

Comment:
It's not needed?

Reply:
`parse` is already required by `parse-server` so not needed. 

> Do you think it's a good idea to include `@cloud`, `@parse` `@schema` aliases?

You just need to put an extra slash when writing @/cloud` etc so I would say it is unnecessary.